state/storage_test.go

```diff
@@ -822,6 +822,56 @@ func TestAllJoinedMembers(t *testing.T) {
 	}
 }
 
+func TestCircularSlice(t *testing.T) {
+	testCases := []struct {
+		name    string
+		max     int
+		appends []int64
+		want    []int64 // these get sorted in the test
```

Comment:
I guess this saves you having to work out orders or mess about with modular arithmetic.

Reply:
Yep